src/billing/BillingAccount/BillingAccountControls.tsx

```diff
@@ -241,11 +242,7 @@ export const BillingAccountControls = (props: BillingAccountControlsProps) => {
                   disabled={!selectedDatasetProjectName || !selectedDatasetName}
                   onClick={async () => {
                     setShowSpendReportConfigurationModal(false);
-                    await updateSpendConfiguration(
-                      billingProject.projectName,
-                      selectedDatasetProjectName,
-                      selectedDatasetName
-                    );
+                    await updateSpendConfiguration();
```

Comment:
The method doesn't declare any arguments and just references these variables directly.